src/Components/Publishing/News/NewsDateHeader.tsx

```diff
@@ -0,0 +1,27 @@
+import moment from "moment"
+import React from "react"
+import styled from "styled-components"
+import { pMedia } from "../../Helpers"
+import { Fonts } from "../Fonts"
+
+interface Props {
+  date: string
+}
+
+export const NewsDateHeader: React.SFC<Props> = props => {
+  const { date } = props
+  const today = new Date()
+  const isToday =
+    moment(date).format("MMM D, YYYY") === moment(today).format("MMM D, YYYY")
+  const hasYear = moment(date).format("YYYY") !== moment(today).format("YYYY")
+  const format = hasYear ? "MMM D, YYYY" : "MMM D"
```

Comment:
Nice, could we move these to https://github.com/artsy/reaction/blob/master/src/Components/Publishing/Constants.ts#L71? 

I know they're not used in other places atm, but it helps to quickly understand what _kinds_ of dates we use and could used to advocate for simplifying them.